sympy/ntheory/generate.py

```diff
@@ -349,6 +348,13 @@ def prime(nth):
         sympy.ntheory.primetest.isprime : Test if n is prime
         primerange : Generate all primes in a given range
         primepi : Return the number of primes less than or equal to n
+
+        References
+        ==========
+
+        - https://en.wikipedia.org/wiki/Prime_number_theorem#Table_of_.CF.80.28x.29.2C_x_.2F_log_x.2C_and_li.28x.29
+        - https://en.wikipedia.org/wiki/Prime_number_theorem#Approximations_for_the_nth_prime_number
+        - https://en.wikipedia.org/wiki/Skewes%27_number
```

Comment:
These references could also be numbered.

Reply:
There are a few more similar instances in `ntheory/factor_.py`.

Reply:
I'll check it again. Have not done the renumbering systematically.